.gitignore

```diff
@@ -1,8 +1,10 @@
 .DS_Store
+.idea
```

Comment:
Thanks for updating .gitignore but can you also update the sample SFTP config in the readme to avoid uploading these files to BGA too?